lib/src/operating_system.dart

```diff
@@ -34,30 +34,30 @@ class OperatingSystem {
   OperatingSystem(this.name, bool matchesNavigator(NavigatorProvider navigator))
       : this._matchesNavigator = matchesNavigator;
 
-  static List<OperatingSystem> _knownSystems = [_mac, _windows, _unix, _linux];
+  static List<OperatingSystem> _knownSystems = [_mac, _windows, _linux, _unix];
```

Comment:
The test for "unix" is whether the version contains "X11", which is also the case on Linux.